codecparsers/Makefile.am

```diff
@@ -1,9 +1,24 @@
 libyami_codecparser_source_c = \
+	$(NULL)
+
+if !BUILD_FASTBOOT
+libyami_codecparser_source_c += \
+	dboolhuff.c \
```

Comment:
this is a part of vp8 decoder

Reply:
The pass-rate of vp8 decoding is not dropped, when this file is not compiled. Maybe we can delete it.